src/cli.ts

```diff
@@ -59,20 +62,18 @@ async function runLinuxInstall(
   return destinationFolder
 }
 
-async function MacOSDeps(file: string): Promise<void> {
-  fs.readFile(file, 'utf-8', function (err, data) {
-    if (err) throw err
-
-    const newValue = data.replace(
-      /install -D/gim,
-      '$(brew --prefix coreutils)/bin/ginstall -D'
-    )
+async function MacOSDeps(file: string, githubToken?: string): Promise<void> {
+  const linuxScript = String(fs.readFileSync(file, 'utf-8'))
```

Comment:
Replaced with synchronous reads/writes for the simplicity (should be okay for GitHub action).

Reply:
Nice, but is it possible to introduce that change in separate PR?

Reply:
Good, also I had the idea to refactor unit tests so that they will use Jest mocks. You already have good smoke test suite, so don't need to make actual calls in unit tests too. What do you think?

Reply:
That would be amazing and very much welcome!